infrastructure/zk/src/hyperchain_wizard.ts

```diff
@@ -70,6 +72,14 @@ async function initHyperchain() {
 
     await init(initArgs);
 
+    // if we used matterlabs/geth network, we need custom ENV file for hyperchain compose parts
+    // This breaks `zk status prover` command, but neccessary for working in isolated docker-network
+    // TODO: Think about better implementation
+    if (useMatterlabsGeth) {
+        wrapEnvModify('ETH_CLIENT_WEB3_URL', 'http://geth:8545');
+        wrapEnvModify('DATABASE_URL', 'postgres://postgres:notsecurepassword@postgres:5432/zksync_local');
```

Comment:
it: maybe worth converting the connectino URL to a constant given it is used in more than one place

Reply:
It's possible, but requires re-write a lot of other code, since we need string with `localhost:5432` for most the part of setup process, but in env file we want `postgres:5432`. Same goes for `geth`